services/skus/controllers.go

```diff
@@ -588,6 +589,47 @@ func CreateOrderCreds(svc *Service) handlers.AppHandler {
 	}
 }
 
+// deleteItemCreds handles requests for deleting credentials for an item.
+func deleteItemCreds(svc *Service) handlers.AppHandler {
+	return func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+		ctx := r.Context()
+		lg := logging.Logger(ctx, "skus.deleteItemCreds")
```

Comment:
suggestion: inline with the new approach pass the module to as the second argument and define the func separately. e.g.

`logging.Logger(ctx, "skus").Str("func", "deleteItemCreds").Logger()`

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/2281/commits/a40fb6cdf80029cc56e66e28a0ede7d70f191587